src/services/link/index.js

```diff
@@ -33,8 +56,8 @@ module.exports = function setup() {
 					}
 				})
 				.catch((err) => {
-					logger.warning(err);
-					res.status(500).send(err);
+					logger.error(err);
+					res.status(500).redirect(`${getFrontendUrl()}/link/invalid`);
```

Comment:
the server should not redirect, it answers invalid instead and the frontend should handle things like redirection or rendering the defined view for this error state